emu/src/render/palette_color.rs

```diff
@@ -0,0 +1,129 @@
+use std::fmt::Display;
+
+use crate::bitwise::Bits;
+
+#[derive(PartialEq, Eq, Clone)]
+pub enum PaletteType {
+    BG,
+    OBJ,
+    FULL,
```

Comment:
What to we use `FULL` for?

Reply:
Ok, I think it is because the documentation says:

> Each BG and OBJ palette RAM may be either split into 16 palettes with 16 colors each, or may be used as a single palette with 256 colors.

So `FULL` is to access both areas as a single palette with 512 colors right? But wouldn't it be better to split this behavior into `BG_FULL` and `OBJ_FULL`?

Correct me if I'm wrong! Thanks!

Reply:
Ah! yeah you are right, I missunderstood the docs.